docs/config.rst

```diff
@@ -1,11 +1,606 @@
 Configuring CumulusCI
 =====================
 
-cumulusci.yml
--------------
+CumulusCI merges multiple `YAML <https://yaml.org/>`_ files that allow for configuration to occur at several distinct scopes. All of these files have the same name, ``cumulusci.yml``, but live in different locations on the file system. (See `configuration scopes`_ for further detail.)
 
-Global overrides
-----------------
+The `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file contains the default configuration stored within CumulusCI itself. This is the location of the standard library of tasks and flows that come with CumulusCI.
```

Comment:
```suggestion
The `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file contains the default configuration stored within CumulusCI itself, which includes the standard library of CumulusCI tasks and flows.
```
Rewording to trim words and fix the subject-verb agreement (singular "library" . . . "that comes with" not plural "tasks and flows" "that come with") Also to avoid saying "this" without a subject (something I keep seeing and flagging).